### ISSUE TITLE
Issue installing 7.4 (and other "newer" Versions)

### ISSUE THREAD

**Issue by FishGames** [cheat-engine/cheat-engine#2245]
Hey, i bet you have heard it a lot by now but I think my issue differs from the rest at least a bit.

First I thought I may have not checked something. Since my PC acted weird lately anyways, I decided to completely reinstall everything; so right now, besides firefox as my browser and discord, I am using a clean installation of Windows.

I am trying to install CE via the [CheatEngine74.exe](https://github.com/cheat-engine/cheat-engine/releases/download/7.4/CheatEngine74.exe)

**What problems occur:**

Just like in issue #2236, #2141 and several others the loading bar takes quite some time. **It says the program is not responding but it just takes a really long time (~10 minutes) to proceed.** I don't know what may be the cause of this, just stating that it's not a major problem.

This is the window that shows right after the loading bar has finished loading.
![grafik](https://user-images.githubusercontent.com/30733996/175093090-29236310-4859-41c4-8437-2117c536e979.png)

After clicking "Next"
![grafik](https://user-images.githubusercontent.com/30733996/175093204-b561b519-6d33-4cc0-afbc-13b62efd26d7.png)

Though this only lasts for a short while as it loses its text
![grafik](https://user-images.githubusercontent.com/30733996/175093665-593c6e47-b067-4f17-a27f-4ca8c2c91e18.png)

After another 10 minutes I get the error message, saying CE could not be installed.
![grafik](https://user-images.githubusercontent.com/30733996/175094215-be294080-9862-4f17-a8a8-cfb6281db729.png)

**What I have tried to solve this problem:**

- I followed step by step the exact process of #1954 up to the point where my screen differed to the one in the video. (Despite having disabled Windows Defender temporarily, I have also added exclusions for both the Download and Temp folder; see attached images)
![grafik](https://user-images.githubusercontent.com/30733996/175097169-b8926525-f9d2-40eb-961a-4feddb38a2bb.png)
![grafik](https://user-images.githubusercontent.com/30733996/175097228-e885b131-5b21-4988-b06c-977b2de66aae.png)
- #2099 does not seem to matter for me, as I can download the Installer but not successfully install CE
- I have tried installing both the english and the german version
- As mentioned here https://github.com/cheat-engine/cheat-engine/issues/2020#issuecomment-1030888759: I also have a stable connection through direct connection to my router

**From here on out I am trying to install other Versions of CE:**
- 7.3 says it installed, but is nowhere to be found on the PC besides a temp folder
![grafik](https://user-images.githubusercontent.com/30733996/175102523-b457db4c-738b-4aad-af9f-0e039d4a5846.png)
![grafik](https://user-images.githubusercontent.com/30733996/175102732-12d6b56c-6ec9-46b9-9584-6fef7832643e.png)
1/2 times it also showed me this error, just like the one in #2020
![grafik](https://user-images.githubusercontent.com/30733996/175105644-0057ace8-9bc0-4b9b-97fc-688b769f5d26.png)
I'm quite certain that it is not a virus since the installation is running on a fresh install of WIndows 10 and I am also not using any kind of VPN
- 7.2 doesn't have an installer
- 7.1 and 7.0 immediately close themselves even before showing a window
- 6.8.3 to 6.8 don't have an installer
- 6.5 is so far the only version that works for me that is also available under this github releases; also since it seems to be a different installer it only took me less than 30 seconds to complete the whole installation process

I would love to know any soultion to this issue and would be glad to help with further details if needed.
I am less active on github but will try my best to check the next few days for updates on this issue, otherwise you can contact me on discord: SomeFishy#5562

**Comment by fade2gray** [comment]
Don't know if this person fixed their issue, but see #2236.

**Comment by FishGames** [comment]
Unfortunately my PC is not able to use wi-fi and I also don't have any other devices besides a smartphone where I could test it out on.

As I don't know much about this: why exactly does 6.5 work but the other versions don't? I heard there was at some point the addition of "bloatware"? I do not really care about that stuff but it might have something to do with those? Sorry for any false ideas, I am unknowing and looking for all possibilities.

Issue #2236 seems to be only about the loading bar not responding, which for me is as simple as waiting for the program to respond after a while.

Are there other ways "installing" CE 7.4? I have tried using the source code and build instructions. This sort of worked but CE didn't really get installed but rather just existed as a program itself. When running the .exe in the bin folder it always gave me 1 error which I don't remember (I would have to try that method again as I did that before reinstalling Windows) and also I could not open any other pre-existing cheat-tables with it.

EDIT: Site just updated for me, I will try out that new suggestion with "App & browser control"

**Comment by FishGames** [comment]
That is unfortuantely the exact same link i was downloading it from earlier, though I will try out downloading the installer onto my phone and uploading it to google drive to see if for whatever reason that fixes it.
I would be very confused though, as I can download the installers themselves
![grafik](https://user-images.githubusercontent.com/30733996/175119156-c37fc0b4-aaa0-4b3a-9426-23c1aa1ca613.png)
I'll be back with updates on this in about 20 minutes

**Comment by FishGames** [comment]
Any more suggestions as to what I can do about this? I'm kinda clueless on my end :/

**Comment by Symbai** [comment]
a) Build it yourself
b) Join CE's Patreon
c) One of the very first google search results: https://www.reddit.com/user/ChucksFeedAndSeed/comments/sw4ttp/cheat_engine_74_adfree_installer/

**Comment by Symbai** [comment]
> Any more suggestions as to what I can do about this

Yes but unfortunately I am not allowed to tell you. Ask Dark Byte for the reason or hope that someone else who IS allowed will tell you.

**Comment by FishGames** [comment]
Very well, I think I'll just leave it at that. Thanks for your help, guys.

**Comment by fade2gray** [comment]
It does seem that your ISP is blocking the rest of the download/installation, couldn't you get a cheap wi-fi dongle, create a hotspot with your mobile and download and instal using your phone's data?

You say you tried building your own, but it's a little more involved than compiling just the executables - see #1851.

**Comment by FishGames** [comment]
Option 1: I guess I could do that but I'm not quite willing to try that out since it's not a guarantee fix. Also I closed this since in my opinion that shouldn't be the solution to the issue (aka change ISP or buy hardware-"upgrades")

Option 2: Yeah I didn't go that far into it, I just followed the build instructions as they are on this github main
![grafik](https://user-images.githubusercontent.com/30733996/175524935-b0e8e643-22e0-443b-8934-18a8c21c0e87.png)

I could use CE 7.4; it worked fine with no crashes whatsoever but it didn't let me open any pre-existing cheat tables. Since I realistically only have the technical skill for following instructions, I didn't try any further fixing that problem. Otherwise I would have probably opened up a different issue as this one here was supposed to be for the newer installers.
